template/src/components/dashboard/sites/DashboardSite.tsx

```diff
@@ -34,19 +33,26 @@ const QueryAndHash = () => {
     );
 };
 
-export const DashboardSite = observer(() => {
+export const DashboardSite = () => {
     const [action, setAction] = React.useState<DialogVariant>("add");
     const [count, setCount] = React.useState(1);
 
+    const logout = useAuthStore((state) => state.logout);
```

Comment:
This is kinda confusing - using state as the function parameter suggests that we deal with a variable, but in this case its a function. Could we agree on using `store` as param name to avoid such implication?

Reply:
I'd prefer sticking with state, since that's the zustand convention: https://github.com/pmndrs/zustand